src/object/obj_internal.h

```diff
@@ -240,6 +240,20 @@ ec_bulk_spec_get_skip(int index, struct ec_bulk_spec *skip_list)
 {
 	return skip_list[index].is_skip;
 }
+
+static inline uint32_t
+ec_rs2cs(uint64_t rs, unsigned short cs_exp)
```

Comment:
this function is unused?

Reply:
No it is not used. I can remove it.  I think this patch may not be viable any longer due to the refactoring of the update/fetch code.